modules/account-assignments/variables.tf

```
@@ -1,6 +1,7 @@
 variable "account_assignments" {
   type = list(object({
     account            = string
+    permission_set_name = string
     permission_set_arn = string
     principal_name     = string
     principal_type     = string
```

```suggestion
    account             = string
    permission_set_name = string
    permission_set_arn  = string
    principal_name      = string
    principal_type      = string
```
Just added spaces to fulfill linting.